### PR TITLE
chore: update readme

### LINKED ISSUE #777: Failed to decode param '/%REACT_APP_META_BACKEND%/meta/clusters'
I followed README instruction and use

```
"dev": "DISABLE_ESLINT_PLUGIN=true TSC_COMPILE_ON_ERROR=true REACT_APP_BACKEND=http://<hostname>:8765 npm start"
```

But got an error

```
URIError: Failed to decode param '/%REACT_APP_META_BACKEND%/meta/clusters'
    at decodeURIComponent (<anonymous>)
    at decode_param (/home/kungasc/ydb-embedded-ui/node_modules/express/lib/router/layer.js:172:12)
    at Layer.match (/home/kungasc/ydb-embedded-ui/node_modules/express/lib/router/layer.js:123:27)
    at matchLayer (/home/kungasc/ydb-embedded-ui/node_modules/express/lib/router/index.js:585:18)
    at next (/home/kungasc/ydb-embedded-ui/node_modules/express/lib/router/index.js:226:15)
    at expressInit (/home/kungasc/ydb-embedded-ui/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/home/kungasc/ydb-embedded-ui/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/kungasc/ydb-embedded-ui/node_modules/express/lib/router/index.js:328:13)
    at /home/kungasc/ydb-embedded-ui/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/kungasc/ydb-embedded-ui/node_modules/express/lib/router/index.js:346:12)
```

Temporary fixed by commenting this [line](https://github.com/ydb-platform/ydb-embedded-ui/blob/fe6ad27e5b598191d8a3b5c3aeba94ff3377a164/public/index.html#L27)

### PR DESCRIPTION
**Pull request by artemmufazalov**
Closes #777

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,26 +44,54 @@ To test new features, you can use ydb version that is currently in testing mode
...
+
+```
+DISABLE_ESLINT_PLUGIN=true
+TSC_COMPILE_ON_ERROR=true
+REACT_APP_BACKEND=http://localhost:8765
+REACT_APP_META_BACKEND=undefined
+META_YDB_BACKEND=undefined
+```
+
+2. Run `npm run start`. Your custom params from `.env` file will be applied
```
Comment (Raubzeug) [inline README.md:61]: Shouldn't be the command `npm run dev`?

File: README.md [hunk 2]
```diff
@@ -43,26 +44,54 @@ To test new features, you can use ydb version that is currently in testing mode
...
-### Custom backend in dev mode
+### Custom configuration in dev mode with .env file
 
-YDB docker represents a single node cluster with only one version, small amount of storage groups, PDisks and VDisks. It may be not enough for development purposes. If you have your own development cluster with sufficient amount of entities, you can run the app in the dev mode with this cluster as backend. To do it, set you host to `REACT_APP_BACKEND` variable in `dev` script. For example:
+You can run the app with your own params by adding `.env` file to project root.
+
+1. Add `.env` file to project root.
+
+```
+DISABLE_ESLINT_PLUGIN=true
```
Comment (Raubzeug) [inline README.md:54]: What do you think if we add `.env.example` to the root of the project?

File: README.md [hunk 3]
```diff
@@ -43,26 +44,54 @@ To test new features, you can use ydb version that is currently in testing mode
...
+
+2. Run `npm run start`. Your custom params from `.env` file will be applied
+
+#### Custom backend in dev mode
+
+YDB docker represents a single node cluster with only one version, small amount of storage groups, PDisks and VDisks. It may be not enough for development purposes. If you have your own development cluster with sufficient amount of entities, you can run the app in the dev mode with this cluster as backend. To do it, alter `REACT_APP_BACKEND` param in your `.env` file:
 
 ```
-"dev": "DISABLE_ESLINT_PLUGIN=true TSC_COMPILE_ON_ERROR=true REACT_APP_BACKEND=http://your-cluster-host:8765 npm start"
+DISABLE_ESLINT_PLUGIN=true
```
Comment (Raubzeug) [inline README.md:68]: It seems that's no need to repeat all variables. Let's here and farther keep only changed variables here?

File: README.md [hunk 4]
```diff
@@ -43,26 +44,54 @@ To test new features, you can use ydb version that is currently in testing mode
...
-DISABLE_ESLINT_PLUGIN=true TSC_COMPILE_ON_ERROR=true REACT_APP_BACKEND= REACT_APP_META_BACKEND=http://your-meta-host:8765 npm start
+DISABLE_ESLINT_PLUGIN=true
+TSC_COMPILE_ON_ERROR=true
+REACT_APP_BACKEND=undefined
+REACT_APP_META_BACKEND=http://your-meta-host:8765
+META_YDB_BACKEND=undefined
 ```
 
-if you need to connect to the meta backend from a server then run the app like this:
+If you need to connect to the meta backend from a server, you need to set `META_YDB_BACKEND ` param in `.env`:
```
Comment (Raubzeug) [inline README.md:87]: `META_YDB_BACKEND ` - redundant space.

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (artemmufazalov) [reply README.md:61]: Not. `npm run dev` already has some params set, it doesn't need `.env` file.

```
 "dev": "DISABLE_ESLINT_PLUGIN=true TSC_COMPILE_ON_ERROR=true REACT_APP_BACKEND=http://localhost:8765 REACT_APP_META_BACKEND=undefined npm start",
```

If you want to run the app with some custom set of variables, you can either override `npm run dev` command, which is not very handy, or add `.env` file and run `npm run start`

So:
1) `npm run dev` - predefined set of variables in command
2) `npm run start` - user's set of variables from `.env` file

File: README.md [hunk 5]
```diff
@@ -43,26 +44,48 @@ To test new features, you can use ydb version that is currently in testing mode
...
-### Custom backend in dev mode
+### Custom configuration in dev mode with .env file
 
-YDB docker represents a single node cluster with only one version, small amount of storage groups, PDisks and VDisks. It may be not enough for development purposes. If you have your own development cluster with sufficient amount of entities, you can run the app in the dev mode with this cluster as backend. To do it, set you host to `REACT_APP_BACKEND` variable in `dev` script. For example:
+You can run the app with your own params by adding `.env` file to project root. There is an example in `.env.example`.
+
+1. Add `.env` file to project root.
+
+```
+DISABLE_ESLINT_PLUGIN=true
```
Comment (Raubzeug) [inline README.md:54]: Let's remove content of .env.example file here and add bash script:
```
\```shell script
cp .env.example .env
\```
```

**Review by Raubzeug — CHANGES_REQUESTED** [review]

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**